lib/main.py

```diff
@@ -319,3 +326,14 @@ def version() -> None:
         version = re.sub(r"^v", "", version)
 
     print("wit {}".format(version))
+
+
+def clean(ws, args) -> None:
```

Comment:
See my comment in the `wit foreach` PR -- Unfortunately `wit-manifest.json` isn't a reliable indicator of package-ness. Iterating through the packages from the wit resolution algorithm is the safest way to do this.